app/src/build.json

```diff
@@ -1,5 +1,5 @@
 {
-  "authenticationType": ["code_assignment", "username", "password"],
+  "authenticationType": ["username", "password"],
```

Comment:
please don't change this, this will causing error for nwmis release

Reply:
you need to mock the buildparams

Reply:
I have revert above change mas @dedenbangkit